class.jetpack-autoupdate.php

```diff
@@ -48,5 +59,68 @@ function autoupdate_core( $update, $item ) {
 		}
 		return $update;
 	}
+
+	function automatic_updates_complete( $results ) {
+		$this->autoupdate_results = $results;
+	}
+
+	private function get_successful_updates( $results = 'plugin' ) {
+		$successful_updates = array();
+
+		if ( ! isset( $this->autoupdate_results[ $results ] ) ) {
+			return $successful_updates;
+		}
+
+		foreach( $this->autoupdate_results[ $results ] as $result ) {
+			if ( $result->result ) {
+				switch( $result ) {
+					case 'theme':
+						$successful_updates[] = $result->item->theme;
+						break;
+					default:
+						$successful_updates[] = $result->item->slug;
+				}
+			}
+		}
+
+		return $successful_updates;
+	}
+
+	function log_results() {
```

Comment:
Would it be worth also using `Jetpack::log()` here to keep a record of the actions locally?